CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 3.14.0 (2020-03-30)
+
+- Added the `enablePointerEvents` option to `SkyOverlayConfig` which enables click events to pass through the overlay's backdrop. [#148](https://github.com/blackbaud/skyux-core/pull/148)
+- Added the `backdropClick` event to the `SkyOverlayInstance` to allow overlay instances to listen for backdrop click events. [#148](https://github.com/blackbaud/skyux-core/pull/148)
+- Added the `isTargetAboveElement()` method to the `SkyCoreAdapterService` to allow SKY UX components to detect if an event target element is contained within another element. [#148](https://github.com/blackbaud/skyux-core/pull/148)
+- Fixed the `closed` event on the `SkyOverlayInstance` to emit when the instance is closed by either the `SkyOverlayService` or the instance itself. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```

Comment:
I'd either delete `the` before both instances of `SkyOverlayInstance`. (Alternately, we could add a noun after `SkyOverlayInstance`, but I'm not sure what that noun would be. Is this a component?)

Also, it seems like `emit` should either be `fire` here or it should be followed by whatever it emits. ... I'd also delete `either`, but that might just be me.